ios/src/TangramMap/TGHttpHandler.h

```diff
@@ -0,0 +1,78 @@
+//
+//  TGHttpHandler.h
+//  tangram
+//
+//  Created by Karim Naaji on 11/23/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ A network request completion callback, called when a download request of `TGHttpHandler`
+ completed an asynchronous request.
+*/
+typedef void(^TGDownloadCompletionHandler)(NSData*, NSURLResponse*, NSError*);
+
+/**
+ A configurable http handler used in a `TGMapViewController`.
+
+ `TGMapViewController` provides a default http handler with the following configuration:
+
+    - cache location: `/tangram_cache`
```

Comment:
This interfaces directly with `NSURLCache`, and it seems to be a subdirectory within the bundle application on iOS:

>In iOS, path is the name of a subdirectory of the application’s default cache directory in which to store the on-disk cache (the subdirectory is created if it does not exist).

c.f: https://developer.apple.com/reference/foundation/urlcache/1415637-init